hpc_launcher/cli/torchrun_hpc.py

```diff
@@ -153,12 +161,14 @@ def main():
         # args.output_script,
         args.setup_only,
         args.color_stderr,
-        args.run_from_launch_dir,
-        (args.save_hostlist or args.verbose),
+        args.dry_run,
+        args.launch_dir != None and (args.save_hostlist or args.verbose),
     )
 
     if jobid:
-        logger.info(f"Job ID: {jobid}")
+        logger.info(f"Job ID: {jobid} launched from {folder_name}")
+        if not args.verbose:
```

Comment:
Shouldn't this be inverted? Also, It will be printed twice.

Reply:
No, I added this so that for batch jobs, the ID is always given back to the user.  Either it is logged, or if logging is not verbose, then just print it.  Also, it shouldn't print twice.